File: app/src/main/java/com/darklycoder/wifitool/MainActivity.java
package com.darklycoder.wifitool;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        initEvents();
    }

    private void initEvents() {
        findViewById(R.id.btn_global).setOnClickListener(v -> startActivity(new Intent(MainActivity.this, GlobalMonitorActivity.class)));

        findViewById(R.id.btn_single).setOnClickListener(v -> startActivity(new Intent(MainActivity.this, SingleMonitorActivity.class)));
    }

}

File: app/src/main/java/com/darklycoder/wifitool/MainActivity.kt
package com.darklycoder.wifitool

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        initEvents()
    }

    private fun initEvents() {
        btn_global.setOnClickListener { startActivity(Intent(this@MainActivity, GlobalMonitorActivity::class.java)) }

        btn_single.setOnClickListener { startActivity(Intent(this@MainActivity, SingleMonitorActivity::class.java)) }
    }

}
